app/2fa/client/lib/2fa.js

```diff
@@ -24,14 +24,15 @@ export class Utils2fa {
 		return err;
 	}
 
-	static overrideLoginMethod(loginMethod, loginArgs, cb, loginMethodTOTP) {
+	static overrideLoginMethod(loginMethod, loginArgs, cb, loginMethodTOTP, emailOrUsername = undefined) {
```

Comment:
Having the param as `undefined` is the same as not having a default value
```suggestion
	static overrideLoginMethod(loginMethod, loginArgs, cb, loginMethodTOTP, emailOrUsername) {
```